packages/client/src/client/App/Trades/TradesState/tableTrades.ts

```diff
@@ -361,21 +361,16 @@ const newTradeId$ = merge(trades$, creditTrades$).pipe(
   scan(
```

Comment:
I think the old code should work - makes more logical sense
From running the old code briefly in local production build, it appears to be failing because we are combining the FX and Credit trades data, so the logic is being run twice, subverting up the booleans ...

Reply:
My suggestion would be to keep trade highlighting completely separate for the asset classes (like the hooks below)
As the code is going to be identical, I would use a function to create the final streams (or hooks)

Reply:
Cool